indico/modules/events/editing/models/file_types.py

```diff
@@ -83,6 +83,17 @@ def __table_args__(cls):
             lazy=True
         )
     )
+    review_conditions = db.relationship(
```

Comment:
I'd move the relationship into the `EditingReviewCondition` model - having it on the other side doesn't really make a technical difference, but IMO it's more closely related to the condition rather than the file type ("mapping conditions to file types" and not "mapping file types to conditions")